src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -32,6 +33,18 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
     error = '';
     dots = 1;
 
+    readonly firstMessageContent = {
+        textContent: "Hi, I'm Iris! I can help you with your programming exercise. You can know more about me here.",
```

Comment:
I guess you want to postpone the translation again as in the last PRs?

Reply:
Thanks for reminding, will discuss this with my teammates.

Reply:
It's not an error message so I think we should translate now